Scripts/RemediationScripts/Remediate-InvalidAADObjectRoleAssignments.ps1

```diff
@@ -53,9 +53,9 @@ function Pre_requisites
     This command would check pre requisites modules to perform remediation.
 	#>
 
-    Write-Host "Required modules are: Az.Resources, AzureAD, Az.Accounts" -ForegroundColor Cyan
+    Write-Host "Required modules are: Az.Resources, AzureAD (or AzureADPreview), Az.Accounts, Az.ResourceGraph" -ForegroundColor Cyan
```

Comment:
Is Az.ResourceGraph really being used?

Reply:
Yes, the module Az.ResourceGraph is being used